content/millennium/faq/common-issues.md

```diff
@@ -0,0 +1,425 @@
+---
+title: Common Issues
+layout: faq
+---
+
+The topics below are the most common issues that we identify when
+validating apps to run on our platform.  We suggest you work through
+each of these before submitting an app for validation.
+
+# Common Issues
+
+* TOC
+{:toc}
+
+## API Usage
+
+### Avoid Multiple Resource Calls
+
+Ensure that your app is efficient in the way it makes API calls.
+Duplicate API calls should be removed to avoid additional performance
+load on the system and improve application load time. As part of the
+technical review process, we will review logs to see if calls are
+identical each time and if they are retrieving the same data set.
+
+
+### Implement Paging
+
+To be compatible with a wide variety of FHIR services and for future-proofing,
+we require that you implement a paging data retrieval scheme
+as defined by the [HL7 FHIR specification](https://www.hl7.org/fhir/dstu2/http.html#paging).
+Your app should handle paging for all resources, even if our FHIR server
+does not support it yet. This ensures your app is compatible with more
+FHIR servers and for ours if we add paging in the future for
+performance.
+
+We've documented the default and max pages sizes for each resouce.
```

Comment:
sp: resouce -> resource

I don't think we've documented this information, and they shouldn't really assume anything since we change the defaults/max as we find out more about performance. They should provide a count if they think their application has a max requirement for performance or visual display reasons (that's what count is - just a max)

Reply:
Some resources still have default and max documented, most no longer have it.  I would recommend cutting the first sentence of this paragraph and updating to: "Queries for some resources can result in large data sets. If you..."